Makefile

```diff
@@ -109,14 +110,19 @@ migrate:  ## perform database migrations
 	$(COMPOSE_RUN) edx_cms python manage.py cms migrate
 .PHONY: migrate
 
-run: \
```

Comment:
keeping `run` running `run-graylog` and `run-keycloak` can be a help ? Renaming it `run-all` or something like this ?